### ISSUE TITLE
Void Spirit new innate ability does not provide any bonuses as it was claimed to be

### ISSUE THREAD

**Issue by Plevien** [ValveSoftware/Dota2-Gameplay#17792]
### Ability name

Void Spirit innate ability

### Description

Void Spirit's new innate ability does not provide any secondary bonuses
![Ekran Görüntüsü (20)](https://github.com/ValveSoftware/Dota2-Gameplay/assets/170598936/04e99239-b668-4f89-a4df-6febd8f605e8)

### Example Match ID (and possibly Timestamp)

7750846259

**Comment by krazanas** [comment]
It actually does provide some of the bonuses. You don't get HP, mana and armor, but you do get regens, magic resistance and attack speed

**Comment by Plevien** [comment]
> It actually does provide some of the bonuses. You don't get HP, mana and armor, but you do get regens, magic resistance and attack speed

Yes noticed late unfortunately. Bonuses effected : Hp Regen, Mana Regen, Armor , Magic Resistance
